package.json

```diff
@@ -128,8 +129,8 @@
     "watch": "node ./scripts/watch.js",
     "lingui:extract": "lingui extract",
     "lingui:compile": "lingui compile",
+    "lingui:build": "[ ! -f src/locales/translations/.git ] && git submodule deinit --all -f; git submodule update --init --recursive --remote",
     "typechain:build": "yarn run typechain --target ethers-v5 --out-dir src/typechain src/abi/*.json src/abi/**/*.json",
-    "preinstall": "[ ! -f src/locales/translations/.git ] && git submodule deinit --all -f; git submodule update --init --recursive --remote",
     "postinstall": "yarn typechain:build && yarn lingui:compile"
   },
```

Comment:
I am not sure if I understand this properly: The preinstall script should run automatically only when you run `yarn install` not when you run `yarn start` for instance. What command makes the preinstall script run too in your case?

Reply:
Yes, just running `yarn install`. But devs do that whenever they add a new package & they probably don't intend to bring in the latest language translations at the same time.

Reply:
Ok understood. There is just the caveat that with my other PR runnning lingui:build at least once would be necessary. But maybe we can change the script so lingui:build runs in the case of yarn install only if the translation directory is empty?